### DISCUSSION TITLE
Stylesheets not working in Github Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22659] [Other Feature Feedback, Questions, & Ideas]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/victorbip/victorbip.github.io" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/10137/181072524-a5e9010e-c353-4abe-a6e2-d4fd2db5bc86.png" class="thumbnail onebox-avatar" width="420" height="420">

<h3><a href="https://github.com/victorbip/victorbip.github.io" target="_blank" rel="nofollow noopener">victorbip/victorbip.github.io</a></h3>

<p>Contribute to victorbip/victorbip.github.io development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>For some reason the JS isn’t working on <a href="https://victorbip.github.io/" rel="nofollow noopener">https://victorbip.github.io/</a></p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Hey, welcome to the <strong>GitHub Support Community</strong>! I’ve noticed that in your CSS stylesheet and JS script you’re calling each starting with <code>assets/</code> instead of <code>./assets/</code>; <code>./assets/</code> will ensure that you’re first defining the current directory and then moving onto the next directory.</p>
<p>To call your JS script, use <code>&lt;script src="./assets/js/script.min.js"&gt;&lt;/script&gt;</code>. To call your CSS stylesheet, use <code>&lt;link rel="stylesheet" href="./assets/css/styles.min.css"&gt;</code>.</p>
<p>Hope this fixes your issue. If you have anymore trouble, don’t hesitate to reply back!</p>
